### PR TITLE
Geospatial support for Snowflake

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
- Supports GEOGRAPHY and GEOMETRY of PostGIS from PostgreSQL to Snowflake - both CDC and QRep
- Supports Postgres' POINT data type to Snowflake
- Adds unique timestamp in prefix for S3 test

### CODE REVIEW

File: flow/model/conversion_avro.go [hunk 1]
```diff
@@ -75,6 +76,7 @@ func GetAvroSchemaDefinition(
 	nullableFields := map[string]bool{}
 
 	for _, qField := range qRecordSchema.Fields {
+		log.Infof("qField name: %s, qField type: %s", qField.Name, qField.Type)
```
Comment (iskakaushik) [inline flow/model/conversion_avro.go:79]: remove log

File: flow/model/column.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package model
+
+type ColumnInformation struct {
```
Comment (iskakaushik) [inline flow/model/column.go:3]: add comments on what column map example would look like

File: flow/connectors/utils/postgres.go [hunk 3]
```diff
@@ -20,3 +23,30 @@ func GetPGConnectionString(pgConfig *protos.PostgresConfig) string {
...
+	defer pool.Close()
+
+	err = pool.QueryRow(ctx, "SELECT typname FROM pg_type WHERE oid = $1", dataType).Scan(&typeName)
+	if err != nil {
+		return qvalue.QValueKindString, fmt.Errorf("failed to query pg_type for custom type handling: %w", err)
+	}
+
+	var qValueKind qvalue.QValueKind
+	switch typeName {
+	case "geometry":
```
Comment (iskakaushik) [inline flow/connectors/utils/postgres.go:44]: "point"?

**Review by iskakaushik — COMMENTED** [review]
lets store the typid to typename in the constructor rather than passing around the ConnStr, and pass around the map as needed.

File: flow/connectors/utils/postgres.go (hunk 3 above)
Comment (Amogh-Bharadwaj) [reply flow/connectors/utils/postgres.go:44]: Point in PostGIS comes under GEOMETRY. Example : `GEOMETRY('POINT(1 2)')`, like `GEOMETRY('LINESTRING(1 2, 3 4)')`
The point data type I'm adding support for in this PR is Postgres' point type

**Pull request merged**